client/www/components/dash/Onboarding.tsx

```diff
@@ -228,7 +228,11 @@ function CreateFirstAppScreen(props: {
               props.onAppNameChange(t);
             }}
           />
-          <Button type="submit" disabled={props.appName.trim().length === 0}>
+          <Button
+            type="submit"
+            loading={props.isLoading}
+            disabled={props.appName.trim().length === 0}

```

Comment:
nit: perhaps instead of `loading`, what if we added this to the `disabled` check? 

Reasoning: afaik if we do this, it will have a very quick 'flash' in the UI

Reply:
You'll get a flash in either case because the disabled state also dims the button. The only difference that I can see is that the `loading` should also show a spinner in your mouse (but doesn't seem to do so in Chrome b/c we set the `disabled` prop on the button when loading.

Reply:
Ah cool! I had an assumption that the text would change or something. Carry on 🫡